docs/playbooks/sql/write.md

```diff
@@ -159,3 +159,36 @@ DELETE FROM my_table;
 </Tabs>
 
 Note that if you want to "delete" a single value in a row but not the whole row itself, you could just `UPDATE` the specific column to `NULL`.
+
+## Multiple tables & atomicity
+
+It's important to understand how transactions work in the context of both the Tableland database and base blockchain. The examples above all demonstrate how to mutate a single table, but what happens if you want to alter multiple tables?
+
+When you write data, a mutation request is passed as a single string that includes one or more statements, separated by a semicolon (`;`). This string _cannot_ touch multiple tables but only a single one. For example, the following query is **invalid** and would fail since multiple statements reference different tables (`my_table`, `other_table`) in a single string:
+
+```sql
+/* A single string, different tables */
+INSERT INTO my_table(id) VALUES(1); INSERT INTO other_table(id) VALUES(2);
+```
+
+If, instead, it was the same table in each statement in the string, it would be valid:
+
+```sql
+/* A single string, same table */
+INSERT INTO my_table(id) VALUES(1); INSERT INTO my_table(id) VALUES(2);
+```
+
+The following set of queries are also **valid** if included in a single transaction and will insert values into two different tables (assuming there aren't issues like invalid syntax or permissions):
+
+```sql
```

Comment:
Might it be easier to demonstrate the different between "Same string" and "separate string" transactions using a JS example using arrays? 

exe: 
`mutate("INSERT INTO my_table(id) VALUES(1); INSERT INTO my_table(id) VALUES(2);", "INSERT INTO other_table(id) VALUES(2);)`

Reply:
Fair point! My goal in this section of the docs is to _not_ introduce clients / languages and keep it agnostic / only SQL guides. But, I think that can work as a sort of pseudocode type of example. Will add.